### PR TITLE
Microapps polkadotjs wallet test

### PR DESCRIPTION
**Pull request by MariuszSzpyt**
- Added Polkadot.js wallet connecting test
- Added component/page handlers like WalletWrapper (new) and WalletConnectModal (similar to old one)
- Removed metamaskExtension setup, but soon Talisman/Enkrypt/Subwallet will be added

### CODE REVIEW

File: utils/frontend/microapps-pages/WalletConnectModal.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    try {
+      await waitForElement(this.driver, elementXpath, 2000);
+      const displayed = await (
+        await this.driver.findElement(By.xpath(elementXpath))
+      ).isDisplayed();
+      return displayed;
+    } catch (Error) {
+      return false;
+    }
+  }
```
Comment (goncer) [inline utils/frontend/microapps-pages/WalletConnectModal.ts:46]: is this code the same as Main.ts?

File: utils/frontend/microapps-pages/WalletWrapper.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    try {
+      await waitForElement(this.driver, elementXpath, 2000);
+      const displayed = await (
+        await this.driver.findElement(By.xpath(elementXpath))
+      ).isDisplayed();
+      return displayed;
+    } catch (Error) {
+      return false;
+    }
+  }
```
Comment (goncer) [inline utils/frontend/microapps-pages/WalletWrapper.ts:73]: same?

**Review by goncer — APPROVED** [review]
minor stuff found, nothing blocker :)

File: utils/frontend/microapps-pages/WalletWrapper.ts (hunk 2 above)
Comment (MariuszSzpyt) [reply utils/frontend/microapps-pages/WalletWrapper.ts]: fixed!

File: utils/frontend/microapps-pages/WalletConnectModal.ts (hunk 1 above)
Comment (MariuszSzpyt) [reply utils/frontend/microapps-pages/WalletConnectModal.ts]: fixed!

File: test/microapps-ui/microapps-ui-main.extensions.test.ts [hunk 3]
```diff
@@ -6,32 +6,80 @@ import { WebDriver } from "selenium-webdriver";
...
-    driver = await DriverBuilder.getInstance(false);
-
+  it("User can connect Polkadot wallet", async () => {
     const mga = new Mangata(driver);
     await mga.go();
-    const sidebar = new Sidebar(driver);
-    const appLoaded = await sidebar.isAppDisplayed();
+    const walletWrapper = new WalletWrapper(driver);
+    const mainPage = new Main(driver);
+    const appLoaded = await mainPage.isAppDisplayed();
```
[the remark this answers is not in the document]
Comment (MariuszSzpyt) [reply test/microapps-ui/microapps-ui-main.extensions.test.ts]: yep, renamed

**Pull request merged**